src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
I heard somewhere, that if we have more than 2-3 arguments, we must move each to the next line. 
But now it looks readable. 
I don't know :) 
Let a mentor will say

Reply:
agree that would be better to move each next to the new line. Doesn't seem to be critical in that case though